Content.Client/Medical/CrewMonitoring/CrewMonitoringWindow.xaml.cs

```diff
@@ -111,7 +112,6 @@ public void ShowSensors(List<SuitSensorStatus> sensors, EntityUid monitor, Entit
             };
 
             deparmentLabel.SetMessage(department);
-            deparmentLabel.StyleClasses.Add(StyleNano.StyleClassTooltipActionDescription);
```

Comment:
oh huh I guessed I missed it while pruning stuff out.

as another note currently `ActionTooltip` contains classes that I'm fairly certain never did anything since the style class was called `tooltipActionTitle` and not `StyleClassTooltipActionTitle`, so I'm leaving a comment noting that.

Reply:
oh `ActionTooltip` is just never used anywhere. That explains it.